cern_access/indico_cern_access/templates/_common.html

```diff
@@ -22,3 +25,26 @@
         period: {{ dates }}
     {%- endtrans %}
 {%- endmacro -%}
+
+{%- macro render_identity_form(form) -%}
+    {{ form_rows(form, orientation='horizontal', fields=('birth_date', 'nationality', 'birth_place')) }}
+    <h2 class="vehicle-section-title"><i class="car-icon"></i>{% trans %}Vehicle Registration{% endtrans %}</h2>
+    {{ form_rows(form, orientation='horizontal', fields=('by_car', 'license_plate')) }}
+
+    <div class="car-plate-message weak-hidden">
+        {% call message_box('warning', fixed_width=true) %}
+            {% trans strong='<strong>'|safe,endstrong="</strong>"|safe -%}
+                If you are bringing your own car, please make sure you type in the
+                {{ strong }}correct{{ endstrong }} license plate number! Otherwise {{ strong }}
```

Comment:
not sure if the placeholders are useful for a simple tag with no args  etc.. AFAIK transifex already hides tags nicely

Reply:
We do it in other places, but yeah, you're right.